R/add_annotation.R

```diff
@@ -53,18 +53,23 @@ add_CellLine_annotation <- function(
     # Assertions:
     checkmate::assert_data_table(dt_metadata)
     checkmate::assert_string(fill, null.ok = TRUE)
-    checkmate::assert_string(externalSource, null.ok = TRUE)
+    checkmate::assert(checkmate::check_string(externalSource, null.ok = TRUE),
```

Comment:
Consider changing to`external_source` for consistency.

Reply:
done